src/Utils/Helpers.ts

```diff
@@ -98,3 +98,32 @@ export function obtainWorkspaceRoot(): string {
 export interface FileSelector {
   onFileSelected(uri: vscode.Uri|undefined): void;
 }
+
+/**
+ * Show an information message to ask users whether to save the unsaved file
+ * @param filepath a full path to the file
+ * @return rejected promises if users don't save the file.
+ */
+export async function askToSaveFile(filepath: string): Promise<void> {
+  const unsavedDocuments = vscode.workspace.textDocuments.filter(td => {
+    if ((td.fileName === filepath) && td.isDirty) {
+      return td;
+    }
+  });
+
+  await Promise.all(unsavedDocuments.map(td => {
+    const title = `Do you want to save the changes you made to ${path.parse(td.fileName).name}?`;
+    const detail = undefined;
+    const ansSave = 'Save';
+    return new Promise((resolve, reject) => {
+      vscode.window.showInformationMessage(title, {detail: detail, modal: true}, ansSave)
```

Comment:
If `unsavedDocuments`'s length is 2, and they are two instance of _one_ text document,
I am afraid that a user would see the same message two times. This might make people feel a bit annoyed.

Reply:
But I think that's what happen we use vscode, for example when I tried to close the workspace, it asks me multiple times about one single document whether to save changes or not.

Reply:
I knew that when we use 'Cfg Edit'r, only one opened 'TextDocument' exists and the changes are synchonized between cfg editor and text editor. However, it's a part of feature, so I believe that it didn't work like that before. Based on that, I thought that some editors don't synchronize its editing material in a view with other text documents - that's why I decided to put 'unsavedDocuments' in an array.

Reply:
I understand what's been explained about `unsavedDocuments`. What I am worry is usability.
If two editor object (let's call this o1 and o2) opens the same file, e.g., `foo.txt` and both are modified at different lines,
`unsavedDocuments` will contains two foo.txt and
user will be asked 
"Do you want to save the changes you made to foo.txt"? -->msg 1 
after clicking 'yes'
user will be asked again
"Do you want to save the changes you made to foo.txt"? --> msg 2 (msg string is same with msg 1)
and the second question might confuse the user because user already click yes.

Or another user, who wants to save not o1 but o2, would have a question, like, "which msg is for o2? msg1 or msg2?"

Reply:
Sure, then let me edit the information message to show only once :-D